internal/pkg/cli/env_init.go

```diff
@@ -192,6 +195,12 @@ func (o *initEnvOpts) Validate() error {
 
 // Ask asks for fields that are required but not passed in.
 func (o *initEnvOpts) Ask() error {
+	if o.appName == "" {
```

Comment:
nit: to be consistent with the other `ask*` methods can we move this if-check within `askAppName`?